test/unit/api-client/api-client.general.spec.ts

```diff
@@ -25,7 +25,7 @@ describe('api-client', () => {
 
     describe('getAccount', () => {
         it('get account email and balance', async () => {
-            const accountEmail = 'test@rev.com';
+            const accountEmail = 'example.com';
```

Comment:
This doesn't make sense to change to a link since the account should be an email - why not just ignore emails in link checker?

Reply:
We should be using the example.com domain name, but we could have made it an email like `test@example.com`